client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
/api/automate_domains is for the top level domains and related actions (see https://github.com/abellotti/manageiq_docs/blob/api_v230/api/reference/automate_domains.adoc) but for general browsing of the automate tree to find state machine endpoints and such you can use /api/automate (see https://github.com/abellotti/manageiq_docs/blob/api_v230/api/reference/automate.adoc).